packages/cfd/src/Constants/cfd-password-modal-constants.tsx

```diff
@@ -1,30 +1,116 @@
-import { localize } from '@deriv/translations';
+import { CFD_PLATFORMS, Jurisdiction } from '@deriv/shared';
 
-export const getWalletCFDInfo = (type: string) => {
+export const getWalletCFDInfo = (
+    type: string,
+    is_demo: boolean,
+    platform = CFD_PLATFORMS.MT5,
+    jurisdiction = Jurisdiction.SVG
+) => {
+    // for DerivX icon
+    if (platform === CFD_PLATFORMS.DXTRADE)
+        return {
+            icon: 'IcRebrandingDxtradeDashboard',
+            card_title: 'Deriv X',
+            title: 'Deriv X',
+            description_title: 'Deriv X',
+        };
+
+    // for MT5 for Malta
+    if (jurisdiction === Jurisdiction.MALTA_INVEST)
+        return {
+            icon: 'IcRebrandingMt5Cfds',
+            card_title: is_demo ? 'MT5 CFDs' : 'MT5 CFDs (Malta)',
+            title: is_demo ? 'CFDs' : 'CFDs (Malta)',
```

Comment:
include **demo** inside of the title
```suggestion
            title: is_demo ? 'CFDs demo' : 'CFDs (Malta)',
```